namespaces/_default_.yml

```diff
@@ -1,17 +1,25 @@
-version: 2019.06.24.0
+version: 2019.09.24.0
 
 field_defaults:
   type: keyword
   doc_values: true
   index: true
 
-_default_:
+# NOTE: the key '_index_type_' must be equal to supported.index_type_placeholder value
```

Comment:
where is `supported.index_type_placeholder` set?  Is the default value `_index_type_`?

Reply:
What happens when we use this file to generate the index templates for ES5?

Reply:
> where is `supported.index_type_placeholder` set?

https://github.com/ViaQ/elasticsearch-templates/pull/112/files#diff-f83f12b083f75074fe1aad23efe54c61R20

> Is the default value `_index_type_`?

It is the only possible value and it get replaced depending on which ES version is the target from the following constants:
https://github.com/ViaQ/elasticsearch-templates/pull/112/files#diff-f83f12b083f75074fe1aad23efe54c61R24-R28

> What happens when we use this file to generate the index templates for ES5?

It should be handled correctly. :-)
First, we have tests that compare generated templates against released templates (and we run it for both ES 2.x and ES 5.x released templates now).
Second, you can try to generate templates from within the `templates` folder, run `make` and then navigate to one of the subfolders that are named in the `Makefile` like this:
```
INDEX_PATTERN_DIRS = openshift,collectd_metrics
```